frontend/src/MainPage/OnLoadUIFolder/Pill.tsx

```diff
@@ -0,0 +1,31 @@
+import { GPT3Logo } from "../svg/GPT3Logo"
+import { GPT4Logo } from "../svg/GPT4Logo"
+
+export const OnLoadUIPill = () => {
+  return (
+    <div className="relative top-6 flex justify-center items-center rounded-xl p-1 bg-gray-100 text-gray-900 dark:bg-gray-900 opacity-95">
```

Comment:
i think this component can still be simplified even more to make it easier to read. maybe we can reformat our HTML tags like such and include descriptive css class names to make it easier to understand?

eg based off of [this](https://gist.github.com/AlexVipond/743647378a22bf282d4b7e460229b5a7).
```
<div
    className="
        outer-pill-wrapper 
        relative top-6 flex justify-center items-center
        rounded-x1 p-1 bg-gray-100 text-gray-900
        dark:bg-gray-900 opacity-95
    "
>
```

i think having that "outer-pill-wrapper" class at the top helps our eyes understand the purpose of this div. we could do this for all of these html elements and maybe it will make it easier to read